docs/Serverless.md

```diff
@@ -0,0 +1,87 @@
+<h1 align="center">Serverless</h1>
+
+Run serverless applications and REST APIs using your existing Fastify application.
+
+### Attention Readers:
+> Fastify is not designed to run on the AWS Lambda platform.
+The Fastify framework is designed to make implementing a traditional HTTP/S server easy.
+The AWS Lambda platform services requests differently than a standard HTTP/S server;
+thus, we cannot guarantee it will work as expected with Fastify.
+Regardless, based on the examples given in this document,
+it is possible to use Fastify in a serverless environment such as AWS Lambda.
+Again, keep in mind that this is not Fastify's intended use case and
+we do not test for such integration scenarios.
```

Comment:
As I said previously, this block needs to refer to serverless, not the lambda platform.

Reply:
✅

Reply:
This still mention AWS Lambda

Reply:
ahhh you mean “such as AWS Lambda”?
It’s just an example, but ok if you want me to remove it...

Reply:
removed